RestaurantReservation.Db/Repositories/CustomerRepository.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db.DataModels;
+
+namespace RestaurantReservation.Db.Repositories
+{
+    public class CustomerRepository
```

Comment:
please consider adding interfaces ( contracts)
to all of your Reps 
applying Dependency Inversion Principle (DIP)," that high-level modules should not depend on low-level modules; both should depend on abstractions."